src/test/java/org/testng/SampleModule.java

```diff
@@ -0,0 +1,12 @@
+package org.testng;
+
+import com.google.inject.Binder;
+
+/**
+ * @author Bartosz Popiela
```

Comment:
Can we have the `@author` tag removed ? For all new deliveries we are following this as a practice.

Reply:
Done.